## Aligner Parameters:
input_clouds:

  # Path, Name, and Type of the Clouds to Align
  cloud_fixed_path:                   20180524-mavic-uav-maize-eschikon 
  cloud_fixed_name:                   point_cloud
  cloud_moving_path:                  20180524-mavic-ugv-maize-eschikon-row3
  cloud_moving_name:                  point_cloud
  relative_scale:	  	      [1,1] 


aligner_params:

  # Params for the ICP algorithm
  max_iter_number:                    50
  verbosity:                          true

